### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sophiecra**
Added testing instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,20 +35,17 @@ emulator or in a physical phone.
 
 # Testing
 
-In 492 you will write automated tests. When you do you will need to add a
-section that explains how to run them.
-
-The unit tests are in `/test/unit`.
-
-The behavioral tests are in `/test/casper/`.
+The unit tests use the regexep '*.test.ts'
```
Comment (cravend) [inline README.md:38]: This isn't the right regex, I don't remember exactly what is but check the files.

File: README.md [hunk 2]
```diff
@@ -35,20 +35,17 @@ emulator or in a physical phone.
...
+The e2e tests are in '/cypress/e2e'.
```
Comment (cravend) [inline README.md:39]: I would call them behavioral tests like he uses.

File: README.md [hunk 3]
```diff
@@ -35,20 +35,17 @@ emulator or in a physical phone.
...
 ## Testing Technology
 
-In some cases you need to install test runners, etc. Explain how.
+Make sure to run 'yarn install' before testing. 
 
 ## Running Tests
 
-Explain how to run the automated tests.
+In one terminal run the application using 'yarn dev'.
+Then in a second terminal run the tests using 'yarn test'. A window will pop up prompting you to choose E2E testing or component testing. Once one is picked, those tests will run.
```
Comment (cravend) [inline README.md:48]: Same here, maybe say E2E (behavioral) or component (unit). I'm not 100% sure how unit tests appear on the test suite.

**Review by cravend — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sophiecra) [reply README.md]: I believe on the test suite it labels them as E2E and component so I went ahead and used parenthesis for clarification

**Review by knguyen423 — APPROVED** [review]

**Review by cravend — APPROVED** [review]
Looks good

**Pull request merged**